bsp/stm32f10x/drivers/stm32f10x_it.c

```diff
@@ -144,6 +145,20 @@ void EXTI4_IRQHandler(void)
 }
 #endif /* RT_USING_LWIP */
 
+#if (defined (RT_USING_CAN)) && !(defined (STM32F10X_CL))
```

Comment:
Good.

By the way,if you are sure to change the stm32f10x_it.c,on your basis I think there is a better way.Change the position of the macro definition judgment.So that you do not need to modify the CAN related code or macro definition when only using the USB function.
```
void USB_LP_CAN1_RX0_IRQHandler(void)
{
    #if (defined (RT_USING_CAN)) && !(defined (STM32F10X_CL))
     CAN1_RX0_IRQHandler();
    #endif
}
void USB_HP_CAN1_TX_IRQHandler(void)
{
    #if (defined (RT_USING_CAN)) && !(defined (STM32F10X_CL))
     CAN1_TX_IRQHandler();
    #endif
}